salt/modules/elasticsearch.py

```diff
@@ -174,7 +174,7 @@ def alias_get(indices=None, aliases=None, hosts=None, profile=None):
     return None
 
 
-def document_create(index, doc_type, body=None, hosts=None, profile=None):
+def document_create(index, doc_type, body=None, id=None, hosts=None, profile=None):
```

Comment:
@msteed Should `id` be added to the end of the kwarg definitions? In case positional arguments are being used elsewhere, we don't want to have to change the API. (I didn't look at when this function was added...just caught my eye.)